app/models/take_medication_reminder.rb

```diff
@@ -0,0 +1,5 @@
+class TakeMedicationReminder < ActiveRecord::Base
+  belongs_to :medication
+  validates_inclusion_of :active, in: [true, false]
+  scope :active, -> { where(active: true) }
+end
```

Comment:
The reminder to take a medication and to refill a medication look basically identical right now and I did think about having them be one reminder model. But I think that once we start enhancing functionality, their scheduling patterns will inherently be pretty different and it felt like we'd almost immediately have to split them apart.